tests/test_cases/test_force_release/test_force_release.py

```diff
@@ -14,18 +14,16 @@
 
 SIM_NAME = cocotb.SIM_NAME.lower()
 SIM_VERSION = cocotb.SIM_VERSION
-LANGUAGE = os.environ["TOPLEVEL_LANG"].lower().strip()
+LANGUAGE = os.getenv("TOPLEVEL_LANG", "verilog").lower().strip()
+VHDL_GPI_INTERFACE = os.getenv("VHDL_GPI_INTERFACE", "")
```

Comment:
Didn't you change the default for `getenv` in your last PR because we are always exporting it now?

Reply:
I just removed that commit, it's not fixing a problem any more.